src/components/cart/OrderSummary/OrderSummary.tsx

```diff
@@ -20,23 +22,17 @@ function OrderSummary({
   const formattedDiscount = useFormattedPrice(discount)
 
   return (
-    <List className="order-summary" data-order-summary>
-      <li>
-        <span>Subtotal ({numberOfItems} products)</span>
-        <span>{useFormattedPrice(subTotal)}</span>
-      </li>
-      {discount > 0 && (
-        <li data-order-summary-discount>
-          <span>Discount</span>
-          <span>-{formattedDiscount}</span>
-        </li>
-      )}
-      <li className="text__title-subsection">
-        <span>Total</span>
-        <span>{useFormattedPrice(total)}</span>
-      </li>
+    <div className={styles.fsOrderSummary}>
+      <UIOrderSummary
+        subtotalLabel={`Subtotal (${numberOfItems} products)`}
+        subtotalValue={useFormattedPrice(subTotal)}
+        discountLabel="Discount"
+        discountValue={discount > 0 ? `-${formattedDiscount}` : null}
```

Comment:
There is a [compile failure](https://github.com/vtex-sites/nextjs.store/runs/8258301955#:~:text=components/cart/OrderSummary/-,OrderSummary,-.tsx%3A30%3A9) here:
<img width="636" alt="CleanShot 2022-09-09 at 11 59 42@2x" src="https://user-images.githubusercontent.com/381395/189380646-c626a97f-62aa-413c-bbde-211a015208dc.png">
```suggestion
        discountValue={discount > 0 ? `-${formattedDiscount}` : undefined}
```

Reply:
👌